flowcraft/generator/recipe.py

```diff
@@ -501,9 +501,11 @@ def __init__(self, *args, **kwargs):
         # The description of the processes
         # [forkable, input_process, output_process]
         self.process_descriptions = {
-            "reads_download": [False, None,"integrity_coverage|seq_typing|patho_typing"],
+            "reads_download": [False, None, "integrity_coverage|seq_typing|patho_typing|seqsero2_reads|stx_seqtyping_reads"],
```

Comment:
So, if I understood this correctly, this should be added to the Innuendo recipe system, right? @bfrgoncalves ?

Reply:
Yes, it will be added to the INNUENDO platform